src/models/media.ts

```diff
@@ -28,6 +28,8 @@ export interface Media {
 
   provider: string
   source?: string
+  providerName?: string
```

Comment:
How does `providerName` get populated? And should it be snake case if it's coming from the API?

I see the API mapping `provider_name` field to `display_name` in the provider serializer: https://github.com/WordPress/openverse-api/blob/a7955c86d43bff504e8d41454f68717d79dd3a44/api/catalog/api/serializers/provider_serializers.py#L11-L14

But not on the media serializer :thinking:

Reply:
Good question, I wonder the same ❓

Reply:
`providerName` is generated when fetching the media item (only when fetching the single item, because I guess search views do not require it?):
https://github.com/WordPress/openverse-frontend/blob/73d0dac020d9b495326109abf7b4dfdd42ab41d2/src/stores/media/index.ts#L343-L351
I guess we could also split the interface into `API Media` and `Local Media` if that makes it clearer in the future.